devtools-ci-index.yaml

```diff
@@ -2193,154 +2193,198 @@
             ci_project: 'devtools'
             ci_cmd: 'cd ee_tests/ && /bin/bash cico_build_deploy.sh'
             timeout: '30m'
-        - 'devtools-test-end-to-end-ts-{test_url}-{test_suite}':
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
             test_url: openshift.io
-            test_suite: runTest
+            test_suite: smokeTest
```

Comment:
What about my other point about keeping the names of Jenkins jobs lower-case only? I.e. `smokeTest` would become either `smoke-test` or simply `smoke`

Reply:
Good suggestion - I just changed the test names to all lower case.